plugins/chat/assets/javascripts/discourse/components/chat-live-pane.js

```diff
@@ -97,6 +91,8 @@ export default class ChatLivePane extends Component {
 
   @action
   updateChannel() {
+    this.loadedOnce = false;
```

Comment:
minor change to ensure we don't show "all messages" loaded before fetching messages first time